### PR TITLE
grafana/toolkit: Add plugin creation task

### LINKED ISSUE #18506: grafana/toolkit: Introduce scaffolding task
`grafana-toolkit plugin:create` - to bootstrap minimal version of Grafana extension

- [ ] Setups dir structure
- [ ] Creates package.json with basic deps and scripts
- [ ] Creates basic configs (tsconfig, lint, prettier etc.)

### PR DESCRIPTION
**Pull request by dprokop**
Closes #18506 

This PR adds `grafana-toolkit plugin:create [name]` task to grafana-toolkit.

It fetches templates from our Github repos

TODO:
- [x] Update Readme
- [x] Validation (name, org name, author required) 

### To test this locally:
1. Go to `<grafana-dir>/packages/grafana-toolkit`
2. Run `yarn link`
3. In folder where you wanna create plugin run `npx grafana-toolkit plugin:create`

### CODE REVIEW

**Comment by ryantxu** [comment]
One big question for our templates is if we want to use general names like 'MyPanel' or 'DataSource' and then just copying them is fine.... or if we want to encourage a domain prefix and have this task replace them:

Rather than: [DataSource.ts](https://github.com/grafana/simple-datasource/blob/master/src/DataSource.ts), maybe 'SimpleDataSource', and the task would replace `Simple` with `MyXyz` in the file name and file

**Comment by dprokop** [comment]
@ryantxu I don't think forcing naming convention is necessary. As a plugin dev I would be surprised if i.e. CI build failed because of naming convention validation. Feels like too much of control. But I might be wrong. Just can't see clear benefit of that. @marcusolsson any opinion on that?

But, in the context of templates, I think we could do something more interesting than [hello from the panel](https://github.com/grafana/simple-react-panel/blob/master/src/module.tsx#L6) and maybe provide default templates with some Getting started guide specific for a template type?

**Review by ryantxu — APPROVED** [review]
LGTM

The templates themselves can improve independently, and we can revisit the idea of a naming convention that gets setup on install.

File: packages/grafana-toolkit/README.md [hunk 1]
```diff
@@ -132,18 +147,18 @@ The styles will be injected via `style` tag during runtime.
...
+In order for Grafana to pickup up you theme stylesheets you need to use `loadPluginCss` from `@grafana/runtime` package. Typically you would do that in the entrypoint of your plugin:
 
 ```ts
 import { loadPluginCss } from '@grafana/runtime';
 
 loadPluginCss({
-  dark: 'plugins/<YOUR-EXTENSION-NAME>/styles/dark.css',
-  light: 'plugins/<YOUR-EXTENSION-NAME>/styles/light.css',
+  dark: 'plugins/<YOUR-PLUGIN-NAME>/styles/dark.css',
+  light: 'plugins/<YOUR-PLUGIN-NAME>/styles/light.css',
```
Comment (ryantxu) [inline packages/grafana-toolkit/README.md]:
```suggestion
  light: 'plugins/<YOUR-PLUGIN-ID>/styles/light.css',
```

File: packages/grafana-toolkit/README.md [hunk 2]
```diff
@@ -132,18 +147,18 @@ The styles will be injected via `style` tag during runtime.
...
-In order for Grafana to pickup up you theme stylesheets you need to use `loadPluginCss` from `@grafana/runtime` package. Typically you would do that in the entrypoint of your extension:
+In order for Grafana to pickup up you theme stylesheets you need to use `loadPluginCss` from `@grafana/runtime` package. Typically you would do that in the entrypoint of your plugin:
 
 ```ts
 import { loadPluginCss } from '@grafana/runtime';
 
 loadPluginCss({
-  dark: 'plugins/<YOUR-EXTENSION-NAME>/styles/dark.css',
-  light: 'plugins/<YOUR-EXTENSION-NAME>/styles/light.css',
+  dark: 'plugins/<YOUR-PLUGIN-NAME>/styles/dark.css',
```
Comment (ryantxu) [inline packages/grafana-toolkit/README.md]:
```suggestion
  dark: 'plugins/<YOUR-PLUGIN-ID>/styles/dark.css',
```

**Pull request merged**